code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -260,8 +261,19 @@ meteor_act
 	if(!affecting)
 		return FALSE
 
+	if(I.can_broad) //Handles broad attacks
+		if(I.forced_broad || (user.a_intent == I_DISARM) || (I.can_swing && I.wielded && (user.a_intent == I_HURT)))
+			var/L[] = BP_ALL_LIMBS
+			var/temp_zone
+			effective_force /= 3
+			L.Remove(hit_zone)
+			for(var/i = 1, i <= 2, i++)
+				temp_zone = pick(L)
+				L.Remove(temp_zone)
+				..(I, user, effective_force, temp_zone)
+
```

Comment:
```suggestion
	if(I.can_broad && (I.forced_broad || (user.a_intent == I_DISARM) || (I.can_swing && I.wielded && (user.a_intent == I_HURT))))
		var/list/L = BP_ALL_LIMBS.Copy()
		effective_force /= 3
		L.Remove(hit_zone)
		for(var/i = 1, i <= 2, i++)
			var/temp_zone = pick(L)
			L.Remove(temp_zone)
			..(I, user, effective_force, temp_zone)

```

Reply:
That's simply not how it works. Read the code, dummy.